### DISCUSSION TITLE
demo site unresponsive

### DISCUSSION THREAD

**Discussion by LennyLip** [gogs/gogs#6712] [General]
is that ok, that the demo site https://try.gogs.io/ returns HTTP 500 error?

**Comment by LennyLip** [comment]
<img width="785" alt="Screenshot 2022-01-07 at 15 16 26" src="https://user-images.githubusercontent.com/955994/148542914-ea020e76-bade-47bc-a73a-ab4b6cf1149a.png">

**Comment by unknwon** [comment]
Thanks for reporting! It is affected by a very noisy neighbor taking up all the disk space, just freed some. 😂

**Reply by TBG-FR** [reply]
I get a 500 error when trying to register on https://try.gogs.io/ , is that normal ? If so, which credentials should I use to login ? Thanks in advance !

**Reply by unknwon** [reply]
@TBG-FR Thanks for reporting! Apparently the disk is full again 😂 Free some again!
